app/src/main/java/com/github/studydistractor/sdp/data/Event.kt

```diff
@@ -0,0 +1,34 @@
+package com.github.studydistractor.sdp.data
+
+/**
+ * Represent an event, an event is a group distraction that can only be done during a specific
+ * time window
+ *
+ * @property eventId id of the event
+ * @property name name of the event
+ * @property description description of the event
+ * @property start time when the event starts
+ * @property end time when the event ends
+ * @property lateParticipation enable late participation (Can users join in the middle of the event)
+ * @property chatId id of the chat that is linked to the event
+ */
+data class Event(

```

Comment:
appart from the eventId, nothing should be optional I think

Reply:
same